bgpd/bgp_snmp.c

```diff
@@ -29,10 +29,101 @@
 #include "bgpd/bgp_snmp_bgp4.h"
 #include "bgpd/bgp_snmp_bgp4v2.h"
 #include "bgpd/bgp_mplsvpn_snmp.h"
+#include "bgpd/bgp_snmp_clippy.c"
+
+uint32_t bgp_snmp_traps_flags;
+
+static int bgp_cli_snmp_traps_config_write(struct vty *vty);
+
+DEFPY (bgp_snmp_traps_rfc4273,
+       bgp_snmp_traps_rfc4273_cmd,
+       "bgp snmp traps rfc4273<enable$on|disable$off>",
```

Comment:
Commands in FRR are usually "something enable" and "no something enable", not "something <enable|disable>". I think it's better to conform to this standard.

Reply:
taken into account